static/src/js/components/PermissionForm/__tests__/PermissionForm.test.jsx

```diff
@@ -278,8 +278,8 @@ describe('PermissionForm', () => {
         await user.type(nameField, 'Test Name')
```

Comment:
Likewise here: lines 69-82 and 342

Reply:
I did not touch the component PermissionForm, just fix the text field component it uses. Not enough knowledge of PermissionForm to add tests there.